addons/hr_attendance/static/src/views/attendance_helper_view.js

```diff
@@ -0,0 +1,34 @@
+import { useService } from "@web/core/utils/hooks";
+
+import { Component, onWillStart, useState } from "@odoo/owl";
+
+export class AttendanceActionHelper extends Component {
+    static template = "hr_attendance.AttendanceActionHelper";
+    static props = ["noContentHelp"];
+    setup() {
+        this.orm = useService("orm");
+        this.actionService = useService("action");
+        this.state = useState({
+            hasDemoData: false,
+        });
+        onWillStart(async () => {
+            this.employeeNumber = await this.orm.searchCount(
```

Comment:
I don't like the fact we make 3 RPC calls to actually always hide a button for real customer databases.

Make an api.model method instead. :)